### DISCUSSION TITLE
Ordering Documents within a Collection

### DISCUSSION THREAD

**Discussion by deepu** [outline/outline#1708] [Self Hosting] [answered]
Is there  a way for me to order Documents within a Collection ?

**Comment by tommoor** [comment, marked as answer]
Not yet, it's a work in progress – this PR just landed last week that rebuilt the sidebar in prep for making it reorderable:
https://github.com/outline/outline/pull/1689

**Comment by mpcreation** [comment]
Hello,
I have the same problem. We really need this feature for our team to be able to use the Outline. Do you have any estimate when this will be available?

Kind regards,
Kuba

**Reply by matbrgz** [reply]
The move works well, you didn't need a drag and drop at all.

**Reply by mpcreation** [reply]
But the move doesn't let you re-order documents inside one category.

For example, you have:
Category / Doc 3
Category / Doc 1 
Category / Doc 2

And you want to make it:
Category / Doc 1
Category / Doc 2 
Category / Doc 3

I see no way to do that :(

**Reply by kenanjapalak** [reply]
You could organise your documents by using numbers when naming them. 

For example

1 Administration / 1.01 Doc
1 Administration / 1.02 Doc
1 Administration / 1.03 Doc
2 Finance / 2.01 Doc
2 Finance / 2.02 Doc

etc.

See https://johnnydecimal.com.

It's not for everyone but it might be a (temporary) solution to your problem.

**Reply by tommoor** [reply]
Due to the complexity this feature has been split into three parts:

- Refactoring landed two weeks ago: #1689
- Moving documents via drag and drop should land this week: https://github.com/outline/outline/pull/1717
- Reordering docs via drag and drop probably another couple of weeks
